tgnnexplainer/xgraph/method/base_explainer_tg.py

```diff
@@ -11,7 +11,7 @@
 
 class BaseExplainerTG(object):
     def __init__(self, model: Union[TGAN, None], model_name: str, explainer_name: str, dataset_name: str, all_events: str, explanation_level: str, device, 
-                verbose: bool = True, results_dir: Optional[str] = None, debug_mode: bool=True) -> None:
+                verbose: bool = True, results_dir: Optional[str] = None, debug_mode: bool=True, threshold_num=20) -> None:
```

Comment:
Small detail: is 25 more correct?

Reply:
Hard to say. Paper says 25 the code has 20 everywhere